### DISCUSSION TITLE
Rate the latest updates of LGSL here!

### DISCUSSION THREAD

**Discussion by tltneon** [tltneon/lgsl#27] [Show and tell]
Well, there is a great place to rate the latest  [releases](https://github.com/tltneon/lgsl/releases). Why not?

**Comment by CoZmicShReddeR** [comment]
Always love LGSL and thank for keeping it alive! Wish there was a Xenforo Fork! :p I am using the HTML Widget with Xenforo to show servers by Zones https://www.cozworld.com
